spec/setup.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Test setup for server-side tests.
+ */
+var chai = require("chai");
+var sinonChai = require("sinon-chai");
+
+// Add chai plugins.
+chai.use(sinonChai);
+
+// Add test lib globals.
+global.expect = chai.expect;
+global.assert = chai.assert;
```

Comment:
I prefer assert but I don't care if you want to mix and match or pick another one. Whatevs